src/Helper/RoleHelper.js

```diff
@@ -0,0 +1,21 @@
+const config = require('config');
+
+module.exports = {
+    hasRoles: function (member) {
+        return member !== null && member.roles.cache.array().length > 0;
+    },
+    isMuted: function (member) {
+        return this.hasRoles(member) && member.roles.cache.has(config.mutedRole);
```

Comment:
```suggestion
        return this.hasRole(member, config.mutedRole);
```

Reply:
I think that with v12, this ported functionality from v11 might be rather pointless actually. I'll see if it can simply be dropped without issues.